radware-alteon/info.json

```diff
@@ -0,0 +1,246 @@
+{
+    "name": "radwareAlteon",
+    "version": "1.0.8",
+    "label": "Radware Alteon",
+    "description": "",
+    "publisher": "Danny",
+    "icon_small_name": "RDWR logo small.png",
+    "icon_large_name": "RDWR logo large.png",
+    "category": "",
+    "configuration": {
+        "fields": [
+            {
+                "title": "URL",
+                "type": "text",
+                "name": "uRL",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "value": "https://192.168.1.1",
+                "tooltip": "the api url for the Radware Alteon"
+            },
+            {
+                "title": "b64EncodedAuth",
+                "type": "password",
+                "name": "b64EncodedAuth",
+                "required": true,
```

Comment:
this param can be removed as in backend you can handle the encoding. or else make it optional param.

Reply:
b64EncodedAuth is the api token for the product it is "mandatory"

Reply:
it is the another word for the api token but in the api documentation it stated as the "b64EncodedAuth", not changing the name because the user might have confusion when inputting the "b64EncodedAuth"